<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="sim_time" default="100.0"/>

	<node name="time_keeper" pkg="primer" type="time_keeper.py" output="screen">
        <param name="sim_time" value="$(arg sim_time)"/>
	</node>

</launch>